.travis.yml

```diff
@@ -65,9 +65,11 @@ matrix:
     fast_finish: true
 
     include:
-        # Try MacOS X
+        # Try MacOS X and Windows
         - os: osx
 
+        - os: windows
```

Comment:
What version of windows does it run on?  We probably want Windows 10, but do we want WIndows 7 as well?

Reply:
The only Windows version supported by travis right now is [Windows Server 1803](https://docs.travis-ci.com/user/reference/windows/#windows-version). I'm not sure this is a problem, though, since the main point is just to provide a smoke test for all Windows systems.